homeassistant/components/device_tracker/nmap_tracker.py

```diff
@@ -25,6 +25,8 @@
 CONF_EXCLUDE = 'exclude'
 # Interval in minutes to exclude devices from a scan while they are home
 CONF_HOME_INTERVAL = 'home_interval'
+CONF_OPTIONS = 'scan_options'
+DEFAULT_OPTIONS = '-F --host-timeout 5s '
```

Comment:
Please remove the space at the end and ensure the code can handle it without the space (seems that way from a quick scan), since you cant add spaces at the end of yaml